### DISCUSSION TITLE
Rect1 in container1 ,rect2 and rect3 in container2 rect3 is holes

### DISCUSSION THREAD

**Discussion by pcbren1** [pixijs/pixijs#7578] [Q&A] [answered]
![34](https://user-images.githubusercontent.com/86771584/124374545-c0be4200-dcce-11eb-9335-899543905939.png)

**Comment by ivanpopelyshev** [comment, marked as answer]
```
layer2rect2.blendMode = PIXI.BLEND_MODES.NORMAL;
const filter = new PIXI.filters.AlphaFilter();
filter.blendMode = PIXI.BLEND_MODES.ADD;
ContainerLayer2.filters = [filter];
```

**Comment by eXponenta** [comment]
ERASE blending working only for framebuffers.
You can't composite forward rendering to screen with ERASE, because blend operation applied onto rendered image.

But when you use filters on container by follow @ivanpopelyshev , it automatically create framebuffer. But this will drop batching process and for huge count can extremely drop performance.
